aten/src/ATen/function_wrapper.py

```diff
@@ -594,6 +607,20 @@ def device_guard(option, dispatch_options, dispatch_tensor):
     return '// DeviceGuard omitted'
 
 
+def named_guard(option, tensors, tensorlists):
+    if not option.get('named_guard', True) or (len(tensors) + len(tensorlists) == 0):
```

Comment:
I think we need somewhere to read about the new option. It looks like aten/src/ATen/native/README.md would be the place to document this?